app/schema/user/user.py

```diff
@@ -73,6 +73,8 @@ class User(BaseUser):
     picture: str
     locations: list[Location]
     is_auth_by_google: bool
+    payplus_customer_uid: str | None
+    payplus_card_uid: str | None
```

Comment:
why do we need to add this info to all pydantic models of user? Does frontend really need this info to function. If not, I'd rather avoid spreading this sensitive data. Moreover, you will send user and card Ids of one users to another users via routes like `get_all_jobs`